src/js/controllers/mapController.ts

```diff
@@ -1622,19 +1622,19 @@ export class MapController {
     const allSources = await setLayerSearchSource();
     const [Search] = await loadModules(['esri/widgets/Search']);
 
-    const searchWidget = new Search({
+    new Search({
       view: this._mapview,
       container: searchRef.current,
       sources: allSources,
     });
 
-    searchWidget.on('search-focus', (e: any) => {
+    /*searchWidget.on('search-focus', (e: any) => {
       const selectedLayer = {
         displayField: e.target.activeSource.displayField,
         layerTitle: e.target.activeSource.layer.title,
       };
       store.dispatch(setSelectedSearchWidgetLayer(selectedLayer));
-    });
```

Comment:
let's remove this if is not needed anymore  but other than that everything looks good!